src/gui/processing/models/qgsmodelviewtoolpan.cpp

```diff
@@ -0,0 +1,89 @@
+/***************************************************************************
+                             qgsmodelviewtoolpan.cpp
+                             ------------------------------------
+    Date                 : March 2020
```

Comment:
Did you see on this one for that long? :wink:

Reply:
Bah. Oh well, not worth the travis rebuild time to fix :stuck_out_tongue: